### ISSUE TITLE
Export only produces a blank page (WSL1)

### ISSUE THREAD

**Issue by yheuhtozr** [slidevjs/slidev#553]
**Describe the bug**
First, thank you for the fix for #397. Now `slidev export` no longer aborts, but instead outputs a blank page PDF (even with the default sample) on the same dev.

**To Reproduce**
1. `yarn create slidev`
2. `yarn add --dev playwright-chromium`
3. `yarn slidev export`
4. you get [this file](https://github.com/slidevjs/slidev/files/8496840/slides-export.pdf)

**Desktop (please complete the following information):**

 - OS: Ubuntu 20.04 (WSL1 on Win11)
 - Browser: N/A
 - Slidev version: 0.30.3

**Comment by twitwi** [comment]
If you have a a `global-top.vue`, you're probably facing this issue https://github.com/slidevjs/slidev/issues/466
A workaround is to rerun the export and the second time it should work.

**Comment by yheuhtozr** [comment]
@twitwi Thank you the advice but I don't think I use the config. The problem just happens with a clean setup, and outputs a blank page as if no error occurred. Retry and you get a same result too.

**Comment by tonai** [comment]
Just tried today and I did  not reproduce the issue.
I am using Ubuntu 20.4 like you.

**Comment by yheuhtozr** [comment]
@tonai Are you running on native Ubuntu or WSL?

**Comment by yheuhtozr** [comment]
@tonai Yes, I suspect that it's a WSL-only problem. I don't have a machine with native Ubuntu so I can't confirm it though...

**Comment by yheuhtozr** [comment]
Seems no longer happening in my environment as of 0.34.3. Closed.

**Comment by yheuhtozr** [comment]
Reopened. This issue seems still there on the **WSL1** environment, but not in WSL2.

**Comment by 66RING** [comment]
same on arch linux

Kernel: 5.15.77-1-lts

package.json:

```json
{
  "name": "ppt",
  "private": true,
  "scripts": {
    "build": "slidev build",
    "dev": "slidev --open",
    "export": "slidev export"
  },
  "dependencies": {
    "@slidev/cli": "0.29.2",
    "@slidev/theme-apple-basic": "^0.20.0",
    "@slidev/theme-default": "*",
    "@slidev/theme-seriph": "*"
  },
  "devDependencies": {
    "playwright-chromium": "^1.27.1"
  }
}
```

**Comment by 66RING** [comment]
fix it after update to slidev v0.36.11

**Comment by FerroEduardo** [comment]
Exporting blank PDF using v0.37.1

- Windows 10 21H2
- WSL 1/Ubuntu 22.04.1 LTS
- Kernel: 4.4.0-19041-Microsoft
- npm: 8.19.2
- Node: v16.18.0

Tried:
1. npm i -D playwright-chromium
2. npx playwright install --with-deps
3. npm run export

```json
{
  "name": "slidev",
  "private": true,
  "scripts": {
    "build": "slidev build",
    "dev": "slidev --open",
    "export": "slidev export"
  },
  "dependencies": {
    "@slidev/cli": "^0.37.1",
    "@slidev/theme-apple-basic": "^0.20.0",
    "@slidev/theme-default": "*",
    "@slidev/theme-seriph": "*"
  },
  "devDependencies": {
    "playwright-chromium": "^1.28.1"
  }
}
```
